pandas/plotting/_core.py

```diff
@@ -864,6 +864,21 @@ def line(self, x=None, y=None, **kwargs):
             The values to be plotted.
             Either the location or the label of the columns to be used.
             By default, it will use the remaining DataFrame numeric columns.
+        color : str, int, array_like, or dict, optional
+            The color for each of the DataFrame's columns. Possible values are:
```

Comment:
what does this do for int?

can you add a vesionchanged tag

Reply:
> what does this do for int?

I'm really sorry, this is in there because I started by copying-and-pasting from the docstring of `scatter` (though I'm not sure it should be in there either and so have raised #31127). I'll check such details more carefully in the future.

Thanks for your review, will address your requested changes